src/components/Modal/index.tsx

```diff
@@ -16,13 +16,25 @@ export type ModalProps = {
   onToggle?: () => void;
   /**
    * Whether or not the top section with the close button should be absolute or not.
-   * If yes. It won't push the content down and the padding should be set on the children component.
+   * If yes. It won't push the content down and the hasPadding should be set on the children component.
    */
   isHeaderAbsolute?: boolean;
   /**
    * A title string
    */
   title?: string;
+  /**
+   * Whether or not close button should be displayed
```

Comment:
Please update this comment to also specify that modal is not closable if this is set.